src/minerva_lib/crop.py

```diff
@@ -0,0 +1,298 @@
+from functools import reduce
+import numpy as np
+from .blend import composite_channel
+from . import skimage_inline as ski
+
+
+def scale_image_nearest_neighbor(source, factor):
+    ''' Resizes an image by a given factor using nearest neighbor pixels.
+
+    Arguments:
+        source: A 2D or 3D numpy array to resize
+        factor: The ratio of `out` shape over `source` shape
+
+    Returns:
+        A numpy array with the resized `source`
+    '''
+
+    if factor <= 0:
+        raise ValueError('Scale factor must be above zero')
+
+    factors = [factor, factor, 1]
+
+    out_shape = [int(round(a * b)) for a, b in zip(source.shape, factors)]
```

Comment:
This is needed such that the output image has the same number of color channels as the input image.